{{cookiecutter.project_slug}}/backend/app/app/api/api_v1/endpoints/users.py

```diff
@@ -58,22 +58,19 @@ def create_user(
 def update_user_me(
     *,
     db: Session = Depends(get_db),
-    password: str = Body(None),
-    full_name: str = Body(None),
-    email: EmailStr = Body(None),
+    db: Session = Depends(get_db),
+    user_in: UserUpdate,
     current_user: DBUser = Depends(get_current_active_user),
 ):
     """
     Update own user.
     """
-    current_user_data = jsonable_encoder(current_user)
-    user_in = UserUpdate(**current_user_data)
-    if password is not None:
-        user_in.password = password
-    if full_name is not None:
-        user_in.full_name = full_name
-    if email is not None:
-        user_in.email = email
+    if user_in.password is None:
+        user_in.password = current_user.password
```

Comment:
Actually, a DBUser does not have a password (thanksfully), but an hashed password 🙈